themes/custom/droopler_theme/scss/components/_d_p_gallery.scss

```diff
@@ -46,6 +46,39 @@ $d-p-gallery-bold-color: $color-troilus !default;
   }
 
   .field--name-field-d-media-image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
Zdaje mi się, że ten field ma własny template, a te style wyglądają jak to samo co mógłbyś ogarnać klasami bootstrapowymi. Nie wiem tak na 100% jakie stosujemy na ogół podejście, może @grzegorz-pietrzak-droptica sie wypowie, ale imo mozemy dorzucic po prostu takie klasy w templatce.
To samo będzie sie tyczyć fragmentu poniżej od `field__item` i diva